### DISCUSSION TITLE
When i try to run npm run dev i get this error ELIFECYCLE  Command failed with exit code 1.

### DISCUSSION THREAD

**Discussion by Joshxing** [vercel/next.js#41733] [Help]
### Verify canary release

- [X] I verified that the issue exists in the latest Next.js canary release

### Provide environment information

when i try to run npm run dev this error pop up 
![image](https://user-images.githubusercontent.com/53200365/197342579-c214b48e-4933-4b11-bc4f-f39a6b590309.png)

### Which example does this report relate to?

next

### What browser are you using? (if relevant)

Chrome

### How are you deploying your application? (if relevant)

No

### Describe the Bug

when i try to run npm run dev i got this error 
![image](https://user-images.githubusercontent.com/53200365/197342611-af7c3681-b6b6-4a55-af54-b8706f401504.png)

### Expected Behavior

![image](https://user-images.githubusercontent.com/53200365/197342719-881eed19-46b8-4971-9298-0d0f8177e35e.png)
i'm stuck for a week in this error

### To Reproduce

how to fix this error 
![image](https://user-images.githubusercontent.com/53200365/197342711-4926d571-1d50-475c-a05b-cbb285095ce7.png)

**Comment by thiswallz** [comment]
have you tried not using pnpm, removing the lock file and using regular node 16/ npm

**Comment by Joshxing** [comment]
> have you tried not using pnpm, removing the lock file and using regular node 16/ npm

I try but still got the error

**Comment by Joshxing** [comment]
I try but i still got the error
